spyder_kernels/customize/spyderpdb.py

```diff
@@ -139,8 +139,6 @@ def interaction(self, frame, traceback):
             self.onecmd('exit')
         else:
             self.setup(frame, traceback)
-            if self.send_initial_notification:
-                self.notify_spyder(frame)
```

Comment:
Why is it necessary to move this to `preloop`?

Reply:
Otherwise the first prompt is not correctly handled by the where command

Reply:
The first prompt is not the result of an interaction, so the initial notification is not sent unless I move this to `preloop`. Otherwise the initial interaction is only sent for the second prompt.